src/router/router.ts

```diff
@@ -114,7 +114,7 @@ export function createRouter(baseGuiPath: string = '/', policyDefinitions: Polic
                 title: 'Gateways',
               },
               props: (route) => ({
-                name: route.query.name,
+                selectedDppName: route.query.dpp,
```

Comment:
I don't think I mind either way, but noticed that this could also be `?gateway` instead of `?dpp=` seeing as its the gateway listing, not a blocker at all completely up to you

Reply:
I’ve changed this to conditionally set the right parameter.